content/docs/studio/selfhosted/configuration/index.md

```diff
@@ -0,0 +1,76 @@
+# Configuration
+
+Studio has a unified YAML configuration file, often referred to as `values.yaml`
+in the documentation.
+
+## Updating the configuration
+
+To update the configuration and reload Studio, follow the instructions according
+to your installation method:
+
+<toggle>
+<tab title="AMI">
+
+1. Connect with SSH to your EC2 instance
+
+```shell
```

Comment:
here and everywhere else - please use `cli` instead of `shell` - it will highlight better, and has other nice features (e.g. not including the $ when you are copy pasting ,etc)

Reply:
@shcheklein do we have a nice way to generically add a button to copy contents to clipboard. it is almost necessary for such installation guides

Reply:
We don't and we had a discussion on this here: https://github.com/iterative/gatsby-theme-iterative/issues/181.